### PR TITLE
feanil/pre scope move

### PR DESCRIPTION
**Pull request by feanil**
- docs: Move from restructureText to Markdown
- fix: Add a warning prior to the package move.

### CODE REVIEW

**Review by brian-smith-tcril — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
+# frontend-build
+
+## Warning: Package Moved
```
Comment (arbrandes) [inline README.md:3]: @abdullahwaheed alerted me to the possibility of having the warning be more "dangerous" looking, such as [here](https://www.npmjs.com/package/babel-plugin-react-intl):

image

Not sure how to do that yet, but we should probably look into it.  We can do that later, though, this is not a blocker for this PR.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+While the Paragon design system will continue to receive updates, these updates
+will no longer be published at the @edx/paragon NPM package. Instead, the
```
Comment (arbrandes) [inline README.md:6]:
```suggestion
will no longer be published at the @edx/frontend-build NPM package. Instead, the
```

**Review by arbrandes — APPROVED** [review]
Approved, save for a copy-paste mistake.

File: README.md (hunk 2 above)
Comment (feanil) [reply README.md:6]: ugh, it didn't load your comment, I'll make a follow-up PR to fix this before I merge the update.

File: README.md (hunk 1 above)
Comment (feanil) [reply README.md:3]: Looks like it's not too bad and I can go back and do that for the packages we've already deprecated: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

**Pull request merged**